buildah/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM centos
+
+RUN \
+  # Install runc first, fail fast if something is wrong.
+  yum -y install wget && \
+  wget -O /bin/runc https://github.com/opencontainers/runc/releases/download/v1.0.0-rc5/runc.amd64 && \
```

Comment:
lets write it to `/usr/bin/runc` to ensure that the yum install below clobbers it